paywall/src/utils/routes.js

```diff
@@ -1,11 +1,12 @@
-import { LOCK_PATH_NAME_REGEXP } from '../constants'
+import { LOCK_PATH_NAME_REGEXP, ACCOUNT_REGEXP } from '../constants'
 
 /**
  * Returns a hash of lockAddress and prefix based on a path.
  * @param {*} path
  */
 export const lockRoute = path => {
-  const match = path.match(LOCK_PATH_NAME_REGEXP)
+  const url = new URL(path, 'http://dummy.com')
```

Comment:
Is this url here as a placeholder as part of the refactor step?

Reply:
I'll add a comment, this is because path is a relative path, and so `new URL` throws with "invalid URL" unless you pass in a base path. Since lockRoute doesn't use base path, I'm using a dummy to prevent the error

Reply:
Let's use at least a URL we like: https://unlock-protocol.com and with a comment  to explain the purpose!

Reply:
decided on https://paywall.unlock-protocol.com